client/src/client.rs

```diff
@@ -14,6 +14,7 @@ use iroha_config::{GetConfiguration, PostConfiguration};
 use iroha_crypto::{HashOf, KeyPair};
 use iroha_data_model::prelude::*;
 use iroha_logger::prelude::*;
+use iroha_telemetry::metrics::Status;
```

Comment:
do you think other metrics would be used by the client in the future?

Reply:
I know for a fact that a lot of the metrics will be useful as stats: e.g. number of users per-domain, so I'm fairly confident.